internal/net/ssl/ssl.go

```diff
@@ -228,7 +228,7 @@ func ConfigureCRL(name string, crl []byte, sslCert *ingress.SSLCert) error {
 		return fmt.Errorf("CRL file %v contains invalid data, and must be created only with PEM formatted certificates", name)
 	}
 
-	_, err := x509.ParseCRL(pemCRLBlock.Bytes)
+	_, err := x509.ParseRevocationList(pemCRLBlock.Bytes)
```

Comment:
Fixing the deprecated func, refer to other golang repo with `x509.ParseRevocationList`: https://github.com/vitessio/vitess/blob/d4c5096ace0762a734fb84eabd8eb044ca67ccbb/go/vt/tlstest/tlstest.go#L315C1-L349C3
> internal/net/ssl/ssl.go:231:12: SA1019: x509.ParseCRL has been deprecated since Go 1.19: Use ParseRevocationList instead. (staticcheck)